corehq/apps/hqwebapp/static/cloudcare/scss/formplayer-webapp/form.scss

```diff
@@ -33,6 +33,9 @@ $group-indent: 15px;
 
   .sticky-submit {
     position: sticky;
+    /* Puts the anchored submit button in front of form elements that use bootstrap's .form-control,
```

Comment:
Is sticky submit only implemented on web apps, not app preview?

Reply:
If I remember correctly there was some logic to exclude it from the preview. https://github.com/dimagi/commcare-hq/pull/34490/files#diff-a0495e536c5711042a5da5dfbcd9c8976f0c009df8e52222479ab2d7f488ed89R50